core/repository/consensus/event_repository.cpp

```diff
@@ -4,39 +4,49 @@
 #include <string>
 #include "../../util/convert_string.hpp"
 #include "../../model/transactions/abstract_transaction.hpp"
+#include "../../consensus/consensus_event.hpp"
 
 namespace repository{
 namespace event {
 
-
-  bool add(const std::string& hash, std::unique_ptr<abstract_transaction::AbstractTransaction> tx){
-    if(tx->getType() == abstract_transaction::TransactionType::message){
+  bool add(const std::string& hash, std::unique_ptr<consensus_event::ConsensusEvent> tx){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The function 'add' is never used.](https://www.codacy.com/app/soramitsu/iroha/file/4255998077/issues/source?bid=3712501&fileBranchId=3831026#l12)